### DISCUSSION TITLE
什么时候可以使用DTk开发套件？

### DISCUSSION THREAD

**Discussion by zhoufan1989** [linuxdeepin/developer-center#5310] [Developer Discussion | 开发者讨论]
你的IDE界面功能不错，但是什么时候可以直接使用deepin的DTK开发套件？或者说什么时候会加入DTK的插件进行使用？

**Comment by zhoufan1989** [comment]
此问题主要是针对sig中的deepin-unioncode，希望此IDE能更快的进行DTK的使用和测试。

**Reply by BLumia** [reply]
deepin-unioncode 项目其实不是已有 SIG 维护的，不过就原始问题而言，deepin-unioncode 本身是可以直接使用 DTK 的，用法上和使用别的有 CMake 支持的库是没有什么区别的。
